### ISSUE TITLE
z-fighting on small bars

### ISSUE THREAD

**Issue by paulmasson** [ladybug-tools/spider-covid-19-viz-3d#4]
@theo-armour instead of using three standard boxes stacked on one another, how about constructing your own rectangular geometry with three divisions on each long side for face coloring. I can do it tomorrow if you don’t first. 

P.S. love the coloring of bars: nice direct indication of the state of each country.

**Comment by theo-armour** [comment]
@paulmasson 

I'm happy you like the colored bars. I don't "see" the construction you suggest. Please do have a go.

**Comment by theo-armour** [comment]
@paulmasson 

My morning brain now understands what you are proposing.

Sure have a go!

**Comment by paulmasson** [comment]
@theo-armour after looking again at the current stable, I like that the boxes are not all the same size, since that indicates the red really extends all the way down. My proposal wouldn't show the same thing, so I'm thinking maybe the best ideas would be to remove top faces of the black and green boxes. That would be easier than creating an entirely new object.

I can work on that, but not until we figure out how my contributions will get merged into your most current dev as well as the stable AT THE SAME TIME!

**Comment by paulmasson** [comment]
Solved by cylinder geometry in latest dev
